### PR TITLE
Keeper optimizations

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Keeper improvement: create request processor only once to improve performance.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit 67661eeec533a1cd29bde81097a84d35c860cb56 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51183/67661eeec533a1cd29bde81097a84d35c860cb56/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
</table>

File: src/Coordination/KeeperStorage.cpp [hunk 1]
```diff
@@ -375,26 +375,26 @@ void KeeperStorage::UncommittedState::applyDelta(const Delta & delta)
         delta.operation);
 }
 
-void KeeperStorage::UncommittedState::addDelta(Delta new_delta)
+void KeeperStorage::UncommittedState::addDelta(const Delta & new_delta)
 {
-    const auto & added_delta = deltas.emplace_back(std::move(new_delta));
```
Comment (pufit) [inline src/Coordination/KeeperStorage.cpp:380]: It is tricky that `addDelta` doesn't properly add deltas. Probably it's good to have a comment about it

**Review by pufit — APPROVED** [review]
LGTM

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
WTF

**Pull request closed without merging**